_sass/_person.sass

```diff
@@ -210,9 +210,20 @@
       img
         width: 100%
 
+
   .person-show-title
     font-size: $base-font-size * 1.2
-    margin-bottom: 0
+    margin-bottom: 0px
+    color: $color-text-white
+    font-family: $header-font-stack
+    @include no-underline(false)
+    &:hover
+      text-decoration: underline // Only underline name
+
+    @include no-focus
```

Comment:
What does this do? Is this why it doesn't get underlined when I <kbd>tab</kbd> to the shows?